TLSharp.Tests/app.config

```diff
@@ -1,15 +1,15 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <appSettings>
-    <add key="ApiHash" value="" />
-    <add key="ApiId" value="" />
-    <add key="NumberToAuthenticate" value="" />
-    <add key="CodeToAuthenticate" value="" />
+    <add key="ApiHash" value=""/>
+    <add key="ApiId" value=""/>
+    <add key="NumberToAuthenticate" value=""/>
+    <add key="CodeToAuthenticate" value=""/>
```

Comment:
why do you change this? please remove this noise

Reply:
This is added from VS2017 because XML syntax is wrong in original file. There is no change here - Only correct spaces to MS defaults

Reply:
Well, you can remove this noise from the PullRequest. To not make this kind of mistake in the future, I recommend using a visualizing-tool for committing, for example `git gui`. This way you know exactly what hunks you're committing.

Reply:
Changed to old file with spaces

Reply:
Changed